src/main/java/com/github/vfyjxf/nee/utils/GuiUtils.java

```diff
@@ -16,106 +20,57 @@
  */
 public class GuiUtils {
 
-    public static boolean isPatternTermExGui(GuiScreen container) {
-        try {
-            Class<?> patternTermExClz = Class.forName("appeng.client.gui.implementations.GuiPatternTermEx");
-            return patternTermExClz.isInstance(container);
-        } catch (ClassNotFoundException e) {
-            return false;
-        }
-    }
-
-    public static boolean isPatternTermExContainer(Container container) {
-        try {
-            Class<?> patternTermExClz = Class.forName("appeng.container.implementations.ContainerPatternTermEx");
-            return patternTermExClz.isInstance(container);
-        } catch (ClassNotFoundException e) {
-            return false;
-        }
-    }
-
-    public static boolean isCraftingSlot(Slot slot) {
-        Container container = Minecraft.getMinecraft().thePlayer.openContainer;
-        if (slot != null) {
-            if (container instanceof ContainerPatternTerm || GuiUtils.isPatternTermExContainer(container)) {
-                IContainerCraftingPacket cct = (IContainerCraftingPacket) container;
-                IInventory craftMatrix = cct.getInventoryByName("crafting");
-                return craftMatrix.equals(slot.inventory);
-            } else {
-                return false;
-            }
-        } else {
-            return false;
-        }
-    }
+    private static final boolean isWirelessCraftingTerminalModLoaded = Loader.isModLoaded(ModIDs.WCT);
 
     public static boolean isGuiWirelessCrafting(GuiScreen gui) {
-
-        try {
-            Class<?> guiWirelessCraftingClass =
-                    Class.forName("net.p455w0rd.wirelesscraftingterminal.client.gui.GuiWirelessCraftingTerminal");
-            return guiWirelessCraftingClass.isInstance(gui);
-        } catch (ClassNotFoundException e) {
-            return false;
-        }
+        if (!isWirelessCraftingTerminalModLoaded) return false;
+        return gui instanceof net.p455w0rd.wirelesscraftingterminal.client.gui.GuiWirelessCraftingTerminal;
```

Comment:
This will make class loading fail if WCT is not present, have you tested this mod to work without it?

Reply:
if the mod isn't loaded it returns false and doesn't reach the part that would throw a class not found exception, I just tested it and it works
![image](https://user-images.githubusercontent.com/57050655/190916778-1e9fe849-62c9-42bd-9123-c8923146463f.png)